### ISSUE TITLE
Visual differentiation for previously watched videos

### ISSUE THREAD

**Issue by nedrichards** [media-explorer/media-explorer#159]
When shown in a folder grid previously watched shows should be visually differentiated from unwatched shows (as well as ordered differently).

This is a really useful feature in boxee when you're watching a series and it would help to eliminate some small scintillas of frustration.

**Comment by semiot** [comment]
Yes, agree that this is an important thing, will try to get Kellie to look at it tomorrow...
